### ISSUE TITLE
NVDA DOES NOT CLOSE CONFIGURATION SETTING

### ISSUE THREAD

**Issue by netblue44** [nvaccess/nvda#8378]
Hi
When using the key stroke NVDA+N or the shortcut keys to open any of the settings or configuration menus, NVDA leaves open a configuration dialog that is not visible and can only be closed by restarting NVDA.
this accurse regardless if a change to a setting is made or not.
For sum reason it seams that NVDA+CONTROL+S and NVDA+CONTROL+A does not cause this to happen.
I have noticed this with the last two next of NVDA.
Using Windows10 April update

**Comment by josephsl** [comment]
Hi, so this is not a problem in NVDA 2018.2 RC and the master snapshots? CC @feerrenrut

**Comment by netblue44** [comment]
I have just tested with those versions and the problem does not seam to 
be pressant

**Comment by ehollig** [comment]
I think this was fixed by #8372. Closing
